notebooks/cleaning_data.ipynb

```diff
@@ -0,0 +1,2892 @@
+{
```

Comment:
<p>the feature-selected data from the <em>update shape </em>process is located in this directory path. Please modify to the <em>processed</em> folder's directory. <strong>not to mention returning the previous file from the "featsel" folder.</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Clustering-Analysis/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>